ray-curator/ray_curator/examples/simple_read_example.py

```diff
@@ -0,0 +1,148 @@
+"""Example text processing pipeline using ray-curator."""
+
+import json
+import os
+from pathlib import Path
+
+try:
+    import pyarrow.parquet as pq
+
+    HAS_PYARROW = True
+except ImportError:
+    HAS_PYARROW = False
+    print("Warning: PyArrow not available. Results will not be saved to Parquet.")
```

Comment:
Add PyArrow as a dependency and remove `HAS_PYARROW`?

Reply:
Good point, though currently pyarrow as impicit dependency both from crossfit and cudf

Reply:
Removed this and added pyarrow as an explicit dependency